bundles/core/org.eclipse.smarthome.core.thing/META-INF/MANIFEST.MF

```diff
@@ -15,6 +15,7 @@ Import-Package: com.google.common.base,
  org.eclipse.smarthome.config.core.dto,
  org.eclipse.smarthome.config.core.status,
  org.eclipse.smarthome.config.core.validation,
+ org.eclipse.smarthome.config.xml,
```

Comment:
Sorry, that won't work. As @sjka mentioned, the framework should also work without xml support and thus core.thing must not have a dependency on config.xml.